src/system_tray.rs

```diff
@@ -105,6 +105,16 @@ impl SystemTray {
     self.0.set_icon(icon)
   }
 
+  /// Set new tray menu title.
+  ///
+  /// ## Platform-specific
+  ///
+  /// Only works on **macOS**
+  #[cfg(target_os = "macos")]
+  pub fn set_title(&mut self, title: &str) {
+    self.0.set_title(title)
+  }
+
```

Comment:
This should be in `SystemTrayExtMacOS` trait and you should make a builder variant for `SystemTrayBuilderExtMacOS`. 

You can find them in `src/platform/macos.rs`